### PR TITLE
Added support for WPRDeskNumber and WPRDeskNumberNotes.

### PR DESCRIPTION
**Pull request by rtucker-mozilla**
Per https://bugzilla.mozilla.org/show_bug.cgi?id=1298453 we need to sync some new attributes for WPR.

### CODE REVIEW

File: templates/edit.php [hunk 1]
```diff
@@ -88,17 +88,24 @@
       </select>
     </td>
   </tr>
+  <?php if(isset($user_data['wprdesknumber'][0]) && $user_data['wprdesknumber'][0] != 'Off-Site') {?>
```
Comment (tofumatt) [inline templates/edit.php:91]: Does the database have both a `null` and an "Off-Site" attribute? They'd mean the same thing, right? 🤔

**Comment by tofumatt** [comment]
r+wc, let me know about the `null` thing and I'll merge it if you don't have access.

File: templates/edit.php (hunk 1 above)
Comment (rtucker-mozilla) [reply templates/edit.php:91]: The data gets sync'd into LDAP and while null & 'Off-Site' do functionally mean the same thing with regard to output in phonebook we don't have control ingress to LDAP to set wprdesknumber == null if wprdesknumber == 'Off-Site'. There's also a good chance that this attribute will be used via external LDAP reporting where the differentiation between null and 'Off-Site' could be important.

**Comment by floatingatoll** [comment]
Merging based on r+wc and observed response.

**Pull request merged**